docs/src/pages/[platform]/connected-components/authenticator/customization/override-function-calls/example.angular.mdx

```diff
@@ -0,0 +1,107 @@
+import { Tabs, Alert } from '@aws-amplify/ui-react';
+import { CURRENT_MAJOR_VERSIONS, PREV_MAJOR_VERSIONS } from '@/data/frameworks';
+
+export let latestVersion = CURRENT_MAJOR_VERSIONS['angular'];
+export let prevVersion = PREV_MAJOR_VERSIONS['angular'];
+
+_app.component.html_
+
+```html{1}
+<amplify-authenticator
+  [services]="services"
+  initialState="signUp"
+>
+  <ng-template
+    amplifySlot="authenticated"
+    let-user="user"
+    let-signOut="signOut"
+  >
+    <h2>Welcome, {{ user.username }}!</h2>
+    <button (click)="signOut()">Sign Out</button>
+  </ng-template>
+</amplify-authenticator>
+
+```
+
+_app.component.ts_
+<Tabs.Container defaultValue={latestVersion}>
+  <Tabs.List>
+    <Tabs.Item value={latestVersion}>ui-angular v{latestVersion} (latest)</Tabs.Item>
+    <Tabs.Item value={prevVersion}>ui-angular v{prevVersion}</Tabs.Item>
+  </Tabs.List>
+  <Tabs.Panel value={latestVersion}>
+    In version {latestVersion}, `Auth` function calls are imported directly as shown below. 
+    ```js{3}
+    import { Component, OnInit } from '@angular/core';
+    import { Amplify } from 'aws-amplify';
+    import { signUp } from 'aws-amplify/auth';
+    import awsExports from './aws-exports';
+
+    @Component({
+      selector: 'sign-up-with-email',
+      templateUrl: 'sign-up-with-email.component.html',
+    })
+    export class SignUpWithEmailComponent implements OnInit {
+      constructor() {
+        Amplify.configure(awsExports);
+      }
+      services = {
+        async handleSignUp(formData: Record<string, any>) {
+          let { username, password, options } = formData;
+          // custom username and email
+          username = username.toLowerCase();
+          const customEmail = options?.userAttributes?.email?.toLowerCase();
+          return signUp({
+            username,
+            password,
+            options: {
+              ...formData?.options,
+              userAttributes: {
+                ...formData?.options?.userAttributes,
+                email: customEmail,
+              },
+            },
+          });
+        },
+      };
+    }
+    ```
+  </Tabs.Panel>
+  <Tabs.Panel value={prevVersion}>
+    <Alert role="none" variation="warning" heading="Sign Up Auto Sign In">
+    If you're overriding `signUp` in **version {prevVersion} or earlier**, you must include the `autoSignIn` key and set `enabled` to true, as shown in the example below.
+    </Alert>
+    In earlier versions, `Auth` function calls are imported from the `Auth` object.
```

Comment:
Instead of "earlier" versions prefer an explicit boundary, for example "In `@aws-amplify/ui-angular` versions previous to <WHATEVER_THE_LAST_VERSION_THAT_USED_JS_v5>" so the documentation remains valid on the next major version release. Same comment for other frameworks where this snippet is used